At the moment, running `itstool` with Python 3.12 spews out a bunch of
`SyntaxWarning`s, polluting the command's stderr (which the code in `ardour`'s
`wscript` captures) and causing a configure failure.

diff --git a/wscript b/wscript
index 39761da5f..b72e2ab75 100644
--- a/wscript
+++ b/wscript
@@ -973,7 +973,7 @@ def configure(conf):
 
     # freedesktop translations needs itstool > 1.0.3 (-j option)
     if Options.options.freedesktop:
-        output = subprocess.Popen("itstool --version", shell=True, stderr=subprocess.STDOUT, stdout=subprocess.PIPE).communicate()[0].splitlines()
+        output = subprocess.Popen("itstool --version", shell=True, stderr=subprocess.DEVNULL, stdout=subprocess.PIPE).communicate()[0].splitlines()
         o = output[0].decode('utf-8')
         itstool = o.split(' ')[0]
         version = o.split(' ')[1].split('.')
